internal/gofumpt.go

```diff
@@ -25,6 +25,8 @@ import (
 // bytes may be collapsed onto a single line.
 const shortLineLimit = 60
 
+var octalIntegerLiteralRegexp = regexp.MustCompile(`\A0[0-7_]+\z`)
```

Comment:
`rxOctalInteger` should be enough. The syntax doesn't have integers anywhere other than literals, and a `rx` prefix is a bit easier to work with, like `errFoo`.

Reply:
Done.